src/ViewEngine.fs

```diff
@@ -19,12 +19,15 @@ namespace Feliz.ViewEngine
 open System
 open System.Text
 
+type EventHandlerType = 
```

Comment:
added this union type for event handler, consumers have to deconstruct this type